parm/sfs/postcntrl_sfs.xml

```diff
@@ -657,21 +663,6 @@
        <scale>6.0</scale>
     </param>
 
-    <param>
```

Comment:
@NeilBarton-NOAA The following three max wind products are requested by CPC. Please keep them.